python-main.js

```diff
@@ -1331,16 +1329,21 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
-        $("#command-connect").click(function () {
-            if ($("#command-connect").length) {
-                doConnect();
-            } else {
-                doDisconnect();
-            }
-        });
-        $("#command-serial").click(function () {
-            doSerial();
-        });
+        if (navigator.usb) {
+            $("#command-serial").click(function () {
+                doSerial();
+            });  
+            $("#command-connect").click(function () {
+                if ($("#command-connect").length) {
+                    doConnect();
+                } else {
+                    doDisconnect();
+                }
+            }); 
+        }else{
+            document.getElementById('command-serial').setAttribute('href',"help.html#WebUSB");
+            document.getElementById('command-connect').setAttribute('href',"help.html#WebUSB");
```

Comment:
If a user clicks this button they will be ask if they want to leave the editor, but it is not clear why. Can we change this for a modal (or an alert, although I don't think an alert can add links), that displays a message and offers a link to the help documentation?

Reply:
If the modal is too dificult at this point we could do a confirm message:

```js
if (confirm(config.translate.confirms.xxxxx))  {
    window.open('https://microbit.org','_blank');
};
```

And the message can be `With WebUSB you can program your micro:bit and connect to the serial console directly from the online editor.\nHowever this feature is not supported on this browser.\nClick Ok to find out more.`